pages/eleitores.jsx

```diff
@@ -1,19 +1,261 @@
+	return (
+		<Stack spacing={2}>
+			<Heading as="h2" size="sm" align="left">
+				{t("filters.priorities.label")}
+			</Heading>
+			<Flex wrap="wrap" gap={1}>
+				<CheckboxGroup>
+					<CustomCheckboxCard
+						name="priorities"
+						value="corruption"
+						label={t("filters.priorities.corruption")}
+					/>
+					<CustomCheckboxCard
+						name="priorities"
+						value="gender"
+						label={t("filters.priorities.gender")}
+					/>
+					<CustomCheckboxCard
+						name="priorities"
+						value="security"
+						label={t("filters.priorities.security")}
+					/>
+					<CustomCheckboxCard
+						name="priorities"
+						value="drugs"
+						label={t("filters.priorities.drugs")}
+					/>
+					<CustomCheckboxCard
+						name="priorities"
+						value="migration"
+						label={t("filters.priorities.migration")}
+					/>
+					<CustomCheckboxCard
+						name="priorities"
+						value="race"
+						label={t("filters.priorities.race")}
+					/>
+					<CustomCheckboxCard
+						name="priorities"
+						value="traditionalPopulationEnvironment"
+						label={t("filters.priorities.traditionalPopulationEnvironment")}
+					/>
+					<CustomCheckboxCard
+						name="priorities"
+						value="healthEducationWork"
+						label={t("filters.priorities.healthEducationWork")}
+					/>
+				</CheckboxGroup>
+			</Flex>
+		</Stack>
+	);
+}
+
+function State({ t }) {
+	const states = [
+		{ value: "sp", label: "SP" },
+		{ value: "rj", label: "RJ" },
+		{ value: "es", label: "ES" },
+	];
+	return (
+		<Stack spacing={2}>
+			<Heading as="h2" size="sm" align="left">
+				{t("filters.state.label")}
+			</Heading>
+
+			<Field
+				name="state"
+				component={ReactSelectAdapter}
+				options={states}
+				placeholder={t("filters.state.label")}
+			/>
+		</Stack>
+	);
+}
+
+export default function EleitoresDashboard({ data }) {
+	const { t } = useTranslation("translation", { keyPrefix: "eleitores" });
+
+	const sleep = (ms) => new Promise((resolve) => setTimeout(resolve, ms));
+
+	const onSubmit = async (data) => {
+		await sleep(300);
+		console.log({ data });
+	};
+
+	return (
+		<>
+			<Head>
+				<title>{t("title")}</title>
+				<meta property="og:title" content={t("title")} key="title" />
+			</Head>
+			<Stack
+				flexDir="column"
+				mb="2"
+				justifyContent="center"
+				alignItems="center"
+				p="1rem"
+				backgroundColor="whiteAlpha.900"
+				boxShadow="md"
+			>
+				<Form
+					onSubmit={onSubmit}
+					render={({ handleSubmit, submitting, submitError, ...rest }) => {
+						console.log(rest.values);
+						return (
+							<Box
+								as="form"
+								onSubmit={handleSubmit}
+								bgColor="white"
+								w={{ base: "85vw", md: "768px" }}
+							>
+								<Stack spacing={8}>
+									<Heading as="h1" size="md" align="left">
+										{t("filters.heading")}
+									</Heading>
+
+									<Identity t={t} />
+									<LGBT t={t} />
+									<Parties t={t} />
+									<Priorities t={t} />
+									<State t={t} />
+
```

Comment:
lets pass this to another comp called "Filters" so we can keep the integration logic in the page, and the rendered comps in the shared folder